File: examples/asyncapi-shopping-cart/shopping-cart/src/main/java/io/example/asyncapi/config/DatabaseConfiguration.java
package io.example.asyncapi.config;

import org.springframework.boot.persistence.autoconfigure.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@Configuration
@EnableJpaRepositories("io.example.asyncapi.shoppingcart")
@EntityScan("io.example.asyncapi.shoppingcart")
@EnableJpaAuditing(auditorAwareRef = "springSecurityAuditorAware")
@EnableTransactionManagement
public class DatabaseConfiguration {

}

File: examples/kustomer-address-jpa/src/main/kotlin/io/zenwave360/examples/kotlin/config/DatabaseConfiguration.kt
package io.zenwave360.examples.kotlin.config

import org.springframework.boot.persistence.autoconfigure.EntityScan
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.data.jpa.repository.config.EnableJpaRepositories
import org.springframework.transaction.annotation.EnableTransactionManagement

@org.springframework.context.annotation.Configuration
@EnableJpaRepositories("io.zenwave360.examples.kotlin")
@EntityScan("io.zenwave360.examples.kotlin")
@EnableJpaAuditing(auditorAwareRef = "springSecurityAuditorAware")
@EnableTransactionManagement
open class DatabaseConfiguration
